ios/Podfile.lock

```diff
@@ -0,0 +1,29 @@
+PODS:
+  - Berbix (3.0.4)
+  - berbix_flutter (0.0.1):
```

Comment:
Is this out of date?

Reply:
I believe it's accurate. Version `0.0.1` of the (local-only) pod spec in the official `berbix_flutter` package corresponds to `3.0.4` of the Berbix SDK.